<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Course_Roll_Pitch</class>
 <widget class="QWidget" name="Course_Roll_Pitch">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>533</width>
    <height>530</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGraphicsView" name="Course_GW">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>300</width>
     <height>300</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>300</width>
     <height>300</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>300</width>
     <height>300</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/images/kompass/Compass.jpg);</string>
   </property>
  </widget>
  <widget class="QGraphicsView" name="Roll_GW">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>320</y>
     <width>300</width>
     <height>200</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>300</width>
     <height>200</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>300</width>
     <height>200</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/images/kompass/Roll.jpg);</string>
   </property>
  </widget>
  <widget class="QGraphicsView" name="Pitch_GW">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>10</y>
     <width>200</width>
     <height>300</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>200</width>
     <height>300</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>200</width>
     <height>300</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/images/kompass/Pitch.jpg);</string>
   </property>
   <property name="backgroundBrush">
    <brush brushstyle="NoBrush">
     <color alpha="255">
      <red>0</red>
      <green>255</green>
      <blue>0</blue>
     </color>
    </brush>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="Course_SB">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>340</y>
     <width>70</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="Pitch_SB">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>370</y>
     <width>70</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="Roll_SB">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>400</y>
     <width>70</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>450</y>
     <width>93</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Error</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
